UPGRADING

```diff
@@ -211,8 +220,8 @@ PHP 8.4 UPGRADE NOTES
     or by invoking their constructor directly.
 
 - PDO_DBLIB:
-  . Fixed setAttribute and getAttribute (SakiTakamachi)
-  . Added PdoDbLib class (danack, kocsismate)
+  . Fixed setAttribute and getAttribute
```

Comment:
This entry is actually a bit vague. I see it corresponds to https://github.com/php/php-src/pull/12793
However, that PR did already add an entry for this in the BC section, so this might not be necessary...

Reply:
Thank you! I removed that line.